flow/packet-headers/icmp.yaml

```diff
@@ -0,0 +1,47 @@
+components:
+  schemas:
+    Flow.Icmp:
+      description: ICMP packet header
+      type: object
+      properties:
+        type:
+          x-field-pattern:
+            x-constants:
+              echo_reply: 0
+              echo_request: 8
+              destination_unreachable: 3
+              source_quench: 4
+              redirect_message: 5
+              time_exceeded: 11
+              parameter_problem: 12
+              timestamp: 13
+              timestamp_reply: 14
+              information_request: 15
+              information_reply: 16
+            description: >-
+              ICMP Type
+            format: integer
+            length: 8
+            default: 4
+            features: [count, metric_group]
+        code:
+          x-field-pattern:
+            description: >-
+              ICMP subtype
+            format: integer
+            length: 8
+            default: 0
+            features: [count, metric_group]
+        checksum:
+          x-field-pattern:
+            description: >-
+              Internet checksum
+            format: integer
+            length: 16
+        rest_of_headers:
```

Comment:
missing default

Reply:
For Checksum any default value is not added.  I verified with the other protocols as well.

Reply:
the default is missing from the rest_of_headers field

Reply:
Added Default.